File: VB.NET/console/ctrl_stmt/Backup/ctrl_stmt/sumofdigit.vb
Module sumofdigit
    Sub main()
        Dim s, n, r As Integer
        s = 0
        'n = 563
        n = CInt(Console.ReadLine())
        While (n > 0)
            r = n Mod 10
            s = s + r
            n = n / 10
        End While
        Console.WriteLine("Sum of number" & s)
    End Sub

End Module

File: csharp/console/progm/CALCULATION/SUM/SUM/sumofdigit.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SUM
{
    class sumofdigit
    {
        public static void Main()
        {
            int n, r, s = 0;
            Console.WriteLine("enter the no");
            n = int.Parse(Console.ReadLine());
            while (n > 0)
            {
                r = n % 10;
                s = s + r;
                n = n / 10;
            }
            Console.WriteLine("sum of digit=" + s);
            Console.ReadKey();
        }

    }
}
